src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -573,7 +578,8 @@ export class NotebookModel extends Disposable implements INotebookModel {
 				let newCell = undefined, tailCell = undefined, partialSource = undefined;
 				let newCellIndex = index;
 				let tailCellIndex = index;
-				let newLinesRemoved: string[] = [];
+				let splitCells: SplitCell[] = [];
+				let newlinesBeforeTailCellContent: string;
```

Comment:
This can just be defined down right before line 655 - no reason to have it all the way up here.